ui/src/views/TableEdit.vue

```diff
@@ -44,6 +45,7 @@ export default class TableCreateView extends Vue {
     }
 
     this.resource = this.table?.pointer ?? null
+    this.resource?.addOut(cc.isObservationTable, this.table?.isObservationTable ?? false)
```

Comment:
Pragmatic approach to handle the problem that cc.isObeservationTable is not a property of a Table